GLOB sdist-make: /code/modules/weko-schema-ui/setup.py
c1 recreate: /code/modules/weko-schema-ui/.tox/c1
c1 setuptools_version is setuptools==57.5.0
c1 installdeps: pytest>=3, pytest-cov, -rrequirements2.txt
ERROR: invocation failed (exit code 1), logfile: /code/modules/weko-schema-ui/.tox/c1/log/c1-2.log
================================== log start ===================================
Obtaining Flask_OAuthlib from git+https://github.com/RCOSDP/flask-oauthlib.git@add-redissentinel#egg=Flask_OAuthlib (from -r requirements2.txt (line 69))
  Cloning https://github.com/RCOSDP/flask-oauthlib.git (to revision add-redissentinel) to ./.tox/c1/src/flask-oauthlib
  Running command git clone --filter=blob:none -q https://github.com/RCOSDP/flask-oauthlib.git /code/modules/weko-schema-ui/.tox/c1/src/flask-oauthlib
  Running command git checkout -b add-redissentinel --track origin/add-redissentinel
  Switched to a new branch 'add-redissentinel'
  Branch 'add-redissentinel' set up to track remote branch 'add-redissentinel' from 'origin'.
  Resolved https://github.com/RCOSDP/flask-oauthlib.git to commit 98eb36e1dfc66256fa7ea62237e9879acb906e9d
  Running command git submodule update --init --recursive -q
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining fpdf from git+https://github.com/RCOSDP/pyfpdf.git@f9b032148283d535cabc7789858081c80de36fef#egg=fpdf (from -r requirements2.txt (line 79))
  Cloning https://github.com/RCOSDP/pyfpdf.git (to revision f9b032148283d535cabc7789858081c80de36fef) to ./.tox/c1/src/fpdf
  Running command git clone --filter=blob:none -q https://github.com/RCOSDP/pyfpdf.git /code/modules/weko-schema-ui/.tox/c1/src/fpdf
  Running command git rev-parse -q --verify 'sha^f9b032148283d535cabc7789858081c80de36fef'
  Running command git fetch -q https://github.com/RCOSDP/pyfpdf.git f9b032148283d535cabc7789858081c80de36fef
  Running command git checkout -q f9b032148283d535cabc7789858081c80de36fef
  Resolved https://github.com/RCOSDP/pyfpdf.git to commit f9b032148283d535cabc7789858081c80de36fef
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-accounts (from -r requirements2.txt (line 94))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-communities (from -r requirements2.txt (line 101))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-db (from -r requirements2.txt (line 104))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-deposit (from -r requirements2.txt (line 105))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-files-rest (from -r requirements2.txt (line 106))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-iiif (from -r requirements2.txt (line 109))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-indexer (from -r requirements2.txt (line 110))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-mail (from -r requirements2.txt (line 113))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-oaiharvester (from -r requirements2.txt (line 115))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-oaiserver (from -r requirements2.txt (line 116))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-oauth2server (from -r requirements2.txt (line 117))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-previewer (from -r requirements2.txt (line 121))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-queues (from -r requirements2.txt (line 123))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-records (from -r requirements2.txt (line 124))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-records-rest (from -r requirements2.txt (line 126))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-resourcesyncclient (from -r requirements2.txt (line 128))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-resourcesyncserver (from -r requirements2.txt (line 129))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-s3 (from -r requirements2.txt (line 131))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining invenio_search from git+https://github.com/RCOSDP/invenio-search.git@cff9744c5dc651893a9c51672c5b8da9adc21e16#egg=invenio_search (from -r requirements2.txt (line 132))
  Cloning https://github.com/RCOSDP/invenio-search.git (to revision cff9744c5dc651893a9c51672c5b8da9adc21e16) to ./.tox/c1/src/invenio-search
  Running command git clone --filter=blob:none -q https://github.com/RCOSDP/invenio-search.git /code/modules/weko-schema-ui/.tox/c1/src/invenio-search
  Running command git rev-parse -q --verify 'sha^cff9744c5dc651893a9c51672c5b8da9adc21e16'
  Running command git fetch -q https://github.com/RCOSDP/invenio-search.git cff9744c5dc651893a9c51672c5b8da9adc21e16
  Running command git checkout -q cff9744c5dc651893a9c51672c5b8da9adc21e16
  Resolved https://github.com/RCOSDP/invenio-search.git to commit cff9744c5dc651893a9c51672c5b8da9adc21e16
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining invenio_search_ui from git+https://github.com/RCOSDP/invenio-search-ui.git@74bd3b2990ff27b39e01b6b31be9a0b5fda3dd0f#egg=invenio_search_ui (from -r requirements2.txt (line 133))
  Cloning https://github.com/RCOSDP/invenio-search-ui.git (to revision 74bd3b2990ff27b39e01b6b31be9a0b5fda3dd0f) to ./.tox/c1/src/invenio-search-ui
  Running command git clone --filter=blob:none -q https://github.com/RCOSDP/invenio-search-ui.git /code/modules/weko-schema-ui/.tox/c1/src/invenio-search-ui
  Running command git rev-parse -q --verify 'sha^74bd3b2990ff27b39e01b6b31be9a0b5fda3dd0f'
  Running command git fetch -q https://github.com/RCOSDP/invenio-search-ui.git 74bd3b2990ff27b39e01b6b31be9a0b5fda3dd0f
  Running command git checkout -q 74bd3b2990ff27b39e01b6b31be9a0b5fda3dd0f
  Resolved https://github.com/RCOSDP/invenio-search-ui.git to commit 74bd3b2990ff27b39e01b6b31be9a0b5fda3dd0f
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/invenio-stats (from -r requirements2.txt (line 134))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining kombu from git+https://github.com/RCOSDP/kombu.git@f204fdf078d5e94393c86693f545e2d011f620f5#egg=kombu (from -r requirements2.txt (line 154))
  Cloning https://github.com/RCOSDP/kombu.git (to revision f204fdf078d5e94393c86693f545e2d011f620f5) to ./.tox/c1/src/kombu
  Running command git clone --filter=blob:none -q https://github.com/RCOSDP/kombu.git /code/modules/weko-schema-ui/.tox/c1/src/kombu
  Running command git rev-parse -q --verify 'sha^f204fdf078d5e94393c86693f545e2d011f620f5'
  Running command git fetch -q https://github.com/RCOSDP/kombu.git f204fdf078d5e94393c86693f545e2d011f620f5
  Running command git checkout -q f204fdf078d5e94393c86693f545e2d011f620f5
  Resolved https://github.com/RCOSDP/kombu.git to commit f204fdf078d5e94393c86693f545e2d011f620f5
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining PyPDF2 from git+https://github.com/RCOSDP/PyPDF2.git@fefc684a3a74aff6f99e5dff24f9b4dd1c95169d#egg=PyPDF2 (from -r requirements2.txt (line 194))
  Cloning https://github.com/RCOSDP/PyPDF2.git (to revision fefc684a3a74aff6f99e5dff24f9b4dd1c95169d) to ./.tox/c1/src/pypdf2
  Running command git clone --filter=blob:none -q https://github.com/RCOSDP/PyPDF2.git /code/modules/weko-schema-ui/.tox/c1/src/pypdf2
  Running command git rev-parse -q --verify 'sha^fefc684a3a74aff6f99e5dff24f9b4dd1c95169d'
  Running command git fetch -q https://github.com/RCOSDP/PyPDF2.git fefc684a3a74aff6f99e5dff24f9b4dd1c95169d
  Running command git checkout -q fefc684a3a74aff6f99e5dff24f9b4dd1c95169d
  Resolved https://github.com/RCOSDP/PyPDF2.git to commit fefc684a3a74aff6f99e5dff24f9b4dd1c95169d
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-accounts (from -r requirements2.txt (line 251))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-admin (from -r requirements2.txt (line 252))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-authors (from -r requirements2.txt (line 253))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-bulkupdate (from -r requirements2.txt (line 254))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-deposit (from -r requirements2.txt (line 255))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-gridlayout (from -r requirements2.txt (line 256))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-groups (from -r requirements2.txt (line 257))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-handle (from -r requirements2.txt (line 258))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-index-tree (from -r requirements2.txt (line 259))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-indextree-journal (from -r requirements2.txt (line 260))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-items-autofill (from -r requirements2.txt (line 261))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-items-ui (from -r requirements2.txt (line 262))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-itemtypes-ui (from -r requirements2.txt (line 263))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-logging (from -r requirements2.txt (line 264))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-records (from -r requirements2.txt (line 265))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-records-ui (from -r requirements2.txt (line 266))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Obtaining file:///code/modules/weko-redis (from -r requirements2.txt (line 267))
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /code/modules/weko-schema-ui/.tox/c1/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/code/modules/weko-redis/setup.py'"'"'; __file__='"'"'/code/modules/weko-redis/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /var/tmp/pip-pip-egg-info-l1ediunh
       cwd: /code/modules/weko-redis/
  Complete output (40 lines):
  WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/pytest-runner/
  WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/pytest-runner/
  WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/pytest-runner/
  WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/babel/
  WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/babel/
  WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/babel/
  WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/babel/
  WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /simple/babel/
  ERROR: Could not find a version that satisfies the requirement Babel>=1.3 (from versions: none)
  ERROR: No matching distribution found for Babel>=1.3
  Traceback (most recent call last):
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/setuptools/installer.py", line 75, in fetch_build_egg
      subprocess.check_call(cmd)
    File "/usr/local/lib/python3.6/subprocess.py", line 311, in check_call
      raise CalledProcessError(retcode, cmd)
  subprocess.CalledProcessError: Command '['/code/modules/weko-schema-ui/.tox/c1/bin/python', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/tmp/tmp5fo7zapy', '--quiet', 'Babel>=1.3']' returned non-zero exit status 1.
  
  The above exception was the direct cause of the following exception:
  
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/code/modules/weko-redis/setup.py", line 98, in <module>
      'Development Status :: 1 - Planning',
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/setuptools/__init__.py", line 152, in setup
      _install_setup_requires(attrs)
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/setuptools/__init__.py", line 147, in _install_setup_requires
      dist.fetch_build_eggs(dist.setup_requires)
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/setuptools/dist.py", line 788, in fetch_build_eggs
      replace_conflicting=True,
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 768, in resolve
      replace_conflicting=replace_conflicting
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 1051, in best_match
      return self.obtain(req, installer)
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 1063, in obtain
      return installer(requirement)
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/setuptools/dist.py", line 844, in fetch_build_egg
      return fetch_build_egg(self, req)
    File "/code/modules/weko-schema-ui/.tox/c1/lib/python3.6/site-packages/setuptools/installer.py", line 77, in fetch_build_egg
      raise DistutilsError(str(e)) from e
  distutils.errors.DistutilsError: Command '['/code/modules/weko-schema-ui/.tox/c1/bin/python', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/var/tmp/tmp5fo7zapy', '--quiet', 'Babel>=1.3']' returned non-zero exit status 1.
  ----------------------------------------
WARNING: Discarding file:///code/modules/weko-redis. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

=================================== log end ====================================
ERROR: could not install deps [pytest>=3, pytest-cov, -rrequirements2.txt]; v = InvocationError("/code/modules/weko-schema-ui/.tox/c1/bin/python -m pip install 'pytest>=3' pytest-cov -rrequirements2.txt", 1)
___________________________________ summary ____________________________________
ERROR:   c1: could not install deps [pytest>=3, pytest-cov, -rrequirements2.txt]; v = InvocationError("/code/modules/weko-schema-ui/.tox/c1/bin/python -m pip install 'pytest>=3' pytest-cov -rrequirements2.txt", 1)
